migration/identity-migration/src/main/java/io/camunda/migration/identity/handler/sm/RoleMigrationHandler.java

```diff
@@ -155,7 +155,16 @@ private Set<CreateAuthorizationRequest> createAuthorizationsForRoleAndPermission
     final Set<CreateAuthorizationRequest> authorizations =
         getAuthorizationsByAudience(audiences, permission, roleId, AuthorizationOwnerType.ROLE);
     if (migrationProperties.isResourceAuthorizationsEnabled()) {
-      authorizations.removeIf(auth -> !RBA_IRRELEVANT_RESOURCE_TYPES.contains(auth.resourceType()));
+      // Check if this is the zeebe-api:write permission
+      final String zeebeWritePermission = audiences.getZeebe() + ":write:*";
```

Comment:
For consistency with the tasklist-permission-specifc code further down below, you could move this into the `StaticEntities` class.

Reply:
done